test/new/db/tools/radiff2

```diff
@@ -50,8 +50,8 @@ NAME=radiff2 unified string comparison
 FILE=-
 CMDS=!!radiff2 -quz ../bins/elf/elf_one_symbol_shdr ../bins/elf/elf_one_symbol_shdr1
 EXPECT=<<EOF
--0x00000000:48656c6c "Hello world!\n"
-+0x00000000:41414141
+-0x00000000:48 65 6c 6c 
```

Comment:
This output looks more like a regression than an improvement

Reply:
The quiet mode supposed not to print string out, isn't it?

Reply:
the string is an useful  hint imho

Reply:
Yeah, but for the first program, the string was hidden in the quiet mode which it's not for the second program. Which doesn't make sense, right?

Btw if user wants strings, they can use "-u" option though.